Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -15,6 +15,7 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
         case defaultEngine
         case quickEngines
         case privateSession
+        case fxSuggestSettings
```

Comment:
```suggestion
        case firefoxSuggestSettings
```

Nit: Since we use `firefoxSuggest` everywhere else, let's not introduce this abbreviation here.